src/main/java/org/bookkeepervisualmanager/bookkeeper/BookkeeperManager.java

```diff
@@ -117,16 +152,46 @@ public LedgerManager getLedgerManager() {
 
     public List<Long> getLedgersForBookie(String bookieId) throws BookkeeperException {
```

Comment:
now we are using the DB to compute this list, no more need to scan the entire ZK tree